addons/dev-helpers/setup-go-env.sh

```diff
@@ -14,6 +14,7 @@ else
   echo 'export PATH=$PATH:/usr/local/go/bin' >> ~/.bashrc
   echo 'export GOPATH=~/gospace' >> ~/.bashrc
   echo 'export PATH=~/gospace/bin:$PATH' >> ~/.bashrc
+  GOPATH=~/gospace
```

Comment:
Not sure if this could have any side effects. If you're looking for something generic, how about:
~~~
SETUP='
export PATH=$PATH:/usr/local/go/bin
export GOPATH=~/gospace
export PATH=~/gospace/bin:$PATH
'
echo "$SETUP" >> ~/.bashrc
eval "$SETUP"
~~~
But it's your decision, obviously.